src/components/faceVerification/standalone/screens/FVFlowError.js

```diff
@@ -1,48 +1,73 @@
 // libraries
-import React from 'react'
+import React, { useCallback, useContext, useEffect } from 'react'
 import { View } from 'react-native'
 
+import { FVFlowContext } from '../context/FVFlowContext'
+
 // components
 import Text from '../../../common/view/Text'
 import { CustomButton, Section, Wrapper } from '../../../common'
+import { BlockingUnsupportedBrowser } from '../../../browserSupport/components/UnsupportedBrowser'
 
 // utils
 import { getDesignRelativeHeight } from '../../../../lib/utils/sizes'
 import { exitApp } from '../../../../lib/utils/system'
-import { openLink } from '../../../../lib/utils/linking'
+import { openLink, redirectTo } from '../../../../lib/utils/linking'
+import { useDialog } from '../../../../lib/dialog/useDialog'
 
 import withStyles from '../theme/withStyles'
 
 const DOCS_URL = 'https://doc.gooddollar/sdk/identity'
 const openDocs = () => openLink(DOCS_URL, '_blank')
 
-const FVFlowError = ({ styles }) => (
-  <Wrapper>
-    <Section style={styles.topContainer} grow>
-      <View style={styles.mainContent}>
-        <View style={styles.descriptionContainer}>
-          <View style={styles.descriptionWrapper}>
-            <Text style={styles.text}>Login information is missing, for instructions please visit: </Text>
-            <Text
-              color={'primary'}
-              fontSize={getDesignRelativeHeight(16)}
-              lineHeight={getDesignRelativeHeight(16)}
-              letterSpacing={0.26}
-              fontFamily="Roboto"
-              fontWeight="bold"
-              textDecorationLine="underline"
-              onPress={openDocs}
-            >{`${DOCS_URL}`}</Text>
+const FVFlowError = ({ styles }) => {
+  const { isWebView, unsupportedCopyUrl, rdu } = useContext(FVFlowContext)
+  const { showDialog } = useDialog()
+
+  const navigateBack = useCallback(() => redirectTo(rdu), [rdu])
+
+  useEffect(() => {
```

Comment:
dont understand why this isnt handle at the same place like the regular fv flow in fvintro?
in fvintro you just use a different onUnsupported callback which redirects back to the rdu

also lets say you still use it here, why are you using showDialog and not the same useCamerasupported hook?

Reply:
See other comment

Reply:
just use the hook
useCameraSupported(onCheck: () => !isWebview, checkOnMount:true.....)
instead of calling showmodal

Reply:
See last commit @sirpy